libse/Subtitle.cs

```diff
@@ -96,21 +96,24 @@ public Paragraph GetParagraphOrDefaultById(string id)
         public SubtitleFormat ReloadLoadSubtitle(List<string> lines, string fileName, SubtitleFormat format)
         {
             Paragraphs.Clear();
-            if (format != null && format.IsMine(lines, fileName))
+            if (format == null)
+            {
+                foreach (SubtitleFormat subtitleFormat in SubtitleFormat.AllSubtitleFormats)
+                {
+                    if (subtitleFormat.IsMine(lines, fileName))
+                    {
+                        subtitleFormat.LoadSubtitle(this, lines, fileName);
+                        _format = subtitleFormat;
+                        return subtitleFormat;
+                    }
+                }
+            }
+            else if (format.IsMine(lines, fileName))

```

Comment:
Will prevent same format to run twice!

If given format is null and **IsMine** returns false just return null.